From c8b8e8bd39d7f8158acbb2b98dd79463b7f4f958 Mon Sep 17 00:00:00 2001
From: William Wu <william.wu@rock-chips.com>
Date: Tue, 8 Jan 2019 20:48:10 +0800
Subject: [PATCH] phy: rockchip-inno-usb2: fix some coding style issues

This patch fix some coding style issues reported by checkpatch.pl.

Change-Id: I6a0163fb849203e722dc0e462ddaf3f96cd080a7
Signed-off-by: William Wu <william.wu@rock-chips.com>
---
 drivers/phy/rockchip/phy-rockchip-inno-usb2.c | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/drivers/phy/rockchip/phy-rockchip-inno-usb2.c b/drivers/phy/rockchip/phy-rockchip-inno-usb2.c
index e97895999e0e..ef746bb7e9cb 100644
--- a/drivers/phy/rockchip/phy-rockchip-inno-usb2.c
+++ b/drivers/phy/rockchip/phy-rockchip-inno-usb2.c
@@ -832,8 +832,8 @@ static const struct phy_ops rockchip_usb2phy_ops = {
 
 /* Show & store the current value of otg mode for otg port */
 static ssize_t otg_mode_show(struct device *device,
-					      struct device_attribute *attr,
-					      char *buf)
+			     struct device_attribute *attr,
+			     char *buf)
 {
 	struct rockchip_usb2phy *rphy = dev_get_drvdata(device);
 	struct rockchip_usb2phy_port *rport = NULL;
@@ -868,8 +868,8 @@ static ssize_t otg_mode_show(struct device *device,
 }
 
 static ssize_t otg_mode_store(struct device *device,
-					       struct device_attribute *attr,
-					       const char *buf, size_t count)
+			      struct device_attribute *attr,
+			      const char *buf, size_t count)
 {
 	struct rockchip_usb2phy *rphy = dev_get_drvdata(device);
 	struct rockchip_usb2phy_port *rport = NULL;
@@ -1602,7 +1602,6 @@ static int rockchip_usb2phy_otg_port_init(struct rockchip_usb2phy *rphy,
 			ret = rockchip_set_vbus_power(rport, true);
 			if (ret)
 				return ret;
-
 		}
 		goto out;
 	}
-- 
2.35.3

